core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
Can we just show an example of adding JS references in the header block?  It seems really strange to have an empty JS file that's just there to tell people to override it.